src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="elements">
+          <label>
+            Surname:
+            <input
+              type="text"
+              autocomplete="off"
+              name="surname"
+            >
+          </label>
+        </div>
+
+        <div class="elements">
+          <label>
+            Name:
+            <input
+              type="text"
+              autocomplete="off"
+              name="name"
+            >
+          </label>
+        </div>
+
+        <div class="elements">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              value="12"
+              min="1"
+              max="100"
+              required
+              name="yearsOld"
+            >
+          </label>
+        </div>
+
+        <div class="elements">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="birthday"
+            >
+          </label>
+        </div>
+
+        <div class="elements">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+            >
+          </label>
+        </div>
+
+      </fieldset>
```

Comment:
Don't add line breaks after last child element
```suggestion
        </div>
      </fieldset>
```

Reply:
Fix that all over the code